gui.py

```diff
@@ -86,6 +86,9 @@ def wrap_in_section(content, section_classes="", container_classes="", div_class
 of +1, for instance, means that the day is warmer than 10% of all above-normal days.</li> 
     <li>The black line represents a running 30-day average. This line is less affected by short-term (1-3 day)
 temperature anomalies.</li>
+    <li>The gray dashed lines at three degrees above and below the “normal” line represent typical daily temperature 
```

Comment:
Switch the quotes to `&ldquo;` and `&rdquo;` everywhere (don't just use the plain `&#34;`, it's not the right symbol for left/right quotes.